testing/server.test.js

```diff
@@ -0,0 +1,44 @@
+const request = require('supertest');
+const app = require('../server/index.js');
+const db = require('../database/index.js');
+
+// Check to see if a GET request is sent upon visiting root path
+describe('Test the root path', () => {
+  test('should response a GET method', () => {
+    return request(app).get('/').then((response) => { 
+      expect(response.statusCode).toBe(200);
+    });
+  });
+});
+
+describe('Test fetching room pics', () => {
+  test('should have the same unique exterior object', () => {
+    db.fetchRoomPics(12, (err, docs) => {
+      if (err) {
+        console.log('error during testing');
+        return;
+      }
+      const expected = {"id":69,"url":"https://s3-us-west-1.amazonaws.com/homesnap/exterior/h12.jpg","home_id":12,"order":1,"caption":"''"};
+      expect(JSON.stringify(docs[0])).toEqual(JSON.stringify(expected));
```

Comment:
I think when you use toEqual you don't have to flatten your objects! It should test for deep equality automatically. But if you tried without stringifying and please correct me haha!